src/index.js

```diff
@@ -37,6 +38,15 @@ const App = (appProps) => {
     );
   }
 
+  const appSpecificShortcuts = importShortcuts(['new', 'edit', 'save', 'expandAllSections', 'collapseAllSections', 'expandOrCollapseAccordion', 'openShortcutModal']);
+
+  const renamedShortcuts = renameShortcutLabels(appSpecificShortcuts,
+    [
+      { 'shortcut': 'new', 'label': 'Create a new widget' },
+      { 'shortcut': 'edit', 'label': 'Edit a widget' },
+      { 'shortcut': 'save', 'label': 'Save form changes' }
```

Comment:
All the labels here need to be translatable strings. You can bring intl via `useIntl` hook and translate them

eg: `{ 'shortcut': 'new', 'label': intl.formatMessage({ id: ui-dashboard.createNewWidget )}`